.gitignore

```diff
@@ -1,3 +1,4 @@
+**.patch
```

Comment:
This is only here to prevent accidentally committing patches in case you were wondering.

Reply:
There's a .git/info/exclude file that acts as a local .gitignore in every git repository.

Reply:
i'll remove this in the next commit and add it there.
thanks for letting me know